web/admin/src/editCell.js

```diff
@@ -84,21 +88,24 @@ export class CellEdit extends Component {
             for (var item of this.props.sessions){
                 if (item.sessionName === named){
                   status = false
```

Comment:
This and the next 3 lines can be replaced with `status = (item.sessionName === this.props.task.sessionName)`

Reply:
change from ```if (item.sessionName === named){``` ?

Reply:
Yes,
```javascript
status = false
if (item.sessionName === this.props.task.sessionName){
  status = true
}
```

can be replaced with `status = (item.sessionName === this.props.task.sessionName)` since they will do the same thing